### ISSUE TITLE
Compilation fatal error: attr/xattr.h: No such file or directory (Fedora 24)

### ISSUE THREAD

**Issue by ldorau** [pmem/pmemfile#252]
On Fedora 24:
```
$ make
....
[ 93%] Building C object tests/preload/CMakeFiles/preload_xattr.dir/xattr/xattr.c.o
/home/ldorau/work/pmemfile/tests/preload/xattr/xattr.c:50:24: fatal error: attr/xattr.h: No such file or directory
 #include <attr/xattr.h>
                        ^
compilation terminated.
tests/preload/CMakeFiles/preload_xattr.dir/build.make:62: recipe for target 'tests/preload/CMakeFiles/preload_xattr.dir/xattr/xattr.c.o' failed
make[2]: *** [tests/preload/CMakeFiles/preload_xattr.dir/xattr/xattr.c.o] Error 1
CMakeFiles/Makefile2:3341: recipe for target 'tests/preload/CMakeFiles/preload_xattr.dir/all' failed
make[1]: *** [tests/preload/CMakeFiles/preload_xattr.dir/all] Error 2
Makefile:138: recipe for target 'all' failed
make: *** [all] Error 2
```

**Comment by ldorau** [comment]
```
$ find /usr -name xattr.h
/usr/include/linux/xattr.h
/usr/include/sys/xattr.h
```

**Comment by GBuella** [comment]
Probably something is just messed up with the include directives in cmake, try grepping for "include.*xattr" in cmake files!

**Comment by GBuella** [comment]
No, the man page really mentions `sys/xattr.h`, so probably that should be included.
http://man7.org/linux/man-pages/man2/setxattr.2.html

**Comment by GBuella** [comment]
It fails on my ubuntu desktop too, how did this get a green travis build?

**Comment by GBuella** [comment]
Ahaa, that is why it worked on travis:
https://github.com/pmem/pmemfile/pull/238/commits/34b9f474721586e8c75fe3019ed9e549424f1ade

**Comment by marcinslusarz** [comment]
Fun, my setxattr man pages says you should include attr/xattr.h, and both attr/xattr.h and sys/xattr.h exist on my system. Attr version belongs to libattr1-dev and sys version belongs to libc6-dev.

**Comment by ldorau** [comment]
Fixed by #253
